ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -386,14 +425,29 @@ function CollectionForm({
                                     direction={{ default: 'column' }}
                                     spaceItems={{ default: 'spaceItemsMd' }}
                                 >
-                                    <Title className="pf-u-mb-xs" headingLevel="h2">
-                                        Attach existing collections
-                                    </Title>
-                                    <p>Extend this collection by attaching other sets.</p>
-                                    <CollectionAttacher
-                                        initialEmbeddedCollections={initialEmbeddedCollections}
-                                        onSelectionChange={onEmbeddedCollectionsChange}
-                                    />
+                                    {isReadOnly ? (
+                                        <>
+                                            <Title className="pf-u-mb-xs" headingLevel="h2">
+                                                Attached collections
```

Comment:
Bravo! Does **Add new collection rules** deserve similar treatment?

Reply:
On the other side of the coin, consistent **Attached collections** heading is better for docs team seems clearer (to me, at least).

Reply:
Good point. How about "Attached collections" in both modes, but conditional rendering of "Extend this collection by attaching other sets." which only applies in the edit modes? I think that would be a good call to action in combination with the "Attach" and "Detach" buttons below.

I'll take a look at the collection rules text too.

Reply:
Yes, indeed.